database/helpers/seedDb.js

```diff
@@ -0,0 +1,10 @@
+var db = require('./../connection.js');
```

Comment:
I ran the seeding script and it works well, nice job!

I'd suggest adding the script directly to package.json, which might be easier for others to find. The command I ran was "node database/helpers/seedDb.js"

Reply:
Good catch!  I added "seed" and "drop" in package.json, and removed the API for "initialize" and "drop".  Whiteboard seed link has also been updated.